src/main/features/response/views/macro/check-and-send/admission.njk

```diff
@@ -17,7 +17,7 @@
 
   {% if type.isOfType(DefendantPaymentType.BY_SET_DATE) %}
     {%
-      set whenWillYouPayLabel = t('{{ whenWillYouPayLabel }}: {{ date }}', {
+      set whenWillYouPayLabel = t('The full amount, no later than {{ date }}', {
```

Comment:
Looks good as quick fix, just remove `whenWillYouPayLabel: whenWillYouPayLabel, ` as it is not longer needed.

Long term we would need to review all places where we print payment options and maybe find consistent way of doing it. I is a bit mess ATM.